manuscript.md

```diff
@@ -12,6 +16,30 @@
 
 ## What is currently limiting our ability to predict network structure?
 
+* As this will probably play a part in up setting the tone/concept for the ms 
```

Comment:
This currently lacks a bit of referencing - you can have a look at the preprint I shared in the github teams, and in particular the Jordano papers from 2016